### ISSUE TITLE
Filter custom post type by custom taxonomy

### ISSUE THREAD

**Issue by stephanmax** [WP-API/node-wpapi#230]
Hi there,
I created a new route with

```coffeescript
site.sectorBriefs = site.registerRoute('wp/v2', 'portfolio/(?P<slug>)')
```

which works well. Now I want to filter this custom post type by a custom taxonomy, pretty much yielding this WP-API URL: `wp/v2/portfolio?filter[portfolio_entries]=sector-brief`.

This is the part I am struggling with. I tried using the `param` (no filtering at all) and `filter` (claims `filter is not a function` although I introduced `filter` in the `params` field on the registered route) method, but it didn't work.

Can someone help out?

**Comment by kadamwhite** [comment]
Hello @stephanmax and thanks for opening this issue! You're not missing anything, there's a bug in the currently released version of the library that prevents filters from being used with custom routes. The release candidate is ready and should be up on NPM within a couple of hours, I'll update this issue when it's published and we can see whether that solves your problem!

**Comment by stephanmax** [comment]
Hi @kadamwhite, any news on this one. Sorry for my impatience, I am just looking at using this plugin in my current sprint and if you tell me, that the fix probably won't get in today, I am happy to work around it.

**Comment by kadamwhite** [comment]
@stephanmax Sorry that I missed your comment earlier today. The good news is that we've released v0.11 of the library this evening, so while I'm sorry we didn't get it in earlier to assist in your sprint, you should be able to

```js
site.sectorBriefs = site.registerRoute('wp/v2', 'portfolio/(?P<slug>)', {
  params: [ 'filter ']
});
```
then `site.sectorBriefs().filter( 'portfolio_entries', 'sector-brief' )` to generate your request. I hope this helps; thank you very much for using node-wpapi, and for opening this issue!

**Comment by kadamwhite** [comment]
As this should be addressed by today's release I've closed the issue, but please comment below if you run into any further trouble and we will continue the conversation.
